manage-images.md

```diff
@@ -0,0 +1,120 @@
+
+| Name                        | Android Version | Ubuntu Version | Support Status     | Available since |
+|-----------------------------|-----------------|----------------|------------|---------------|
+| `bionic:android11:amd64`      | 11              | 18.04          | supported | 1.10 |
+| `bionic:android11:arm64`      | 11              | 18.04          | supported | 1.10 |
+| `bionic:android10:amd64`      | 10              | 18.04          | supported | 1.0
+| `bionic:android10:arm64`      | 10              | 18.04          | supported | 1.0
+| `bionic:android7:amd64`       | 7               | 18.04          | unsupported as of 1.10 | 1.0 |
+| `bionic:android7:arm64`       | 7               | 18.04          | unsupported as of 1.10 | 1.0 |
+
+The `bionic:android7:arm64` and `bionic:android7:amd64` images are not receiving Android security updates anymore as Google stopped supporting Android 7 in 2019. They are only provided for legacy reasons and have been dropped with Anbox Cloud 1.10 in April 2021.
+
+## Configure image server
+
+Using the image server hosted by Canonical enables automated updates on your Anbox Cloud deployment. AMS will automatically synchronise new image versions in regular intervals and update your applications. Using the image server should be preferred over using manually managed images. Images on the image server are updated for important security updates or bug fixes, and with every release of Anbox Cloud.
+
+Starting with Anbox Cloud 1.9, access to the image server is automatically configured as part of your Ubuntu Advantage subscription during the charm deployment. No further manual steps are necessary.
+
+AMS will start automatically to import images available on the image server. The `images.update_interval` configuration option allows to customise how often AMS looks for new images. You can set it to a desired interval with the following command:
+
+    amc config set images.update_interval 5m
+
+Synchronised images will become available next to images you manually add:
+
+```bash
++----------------------+-----------------------------+--------+----------+
+|          ID          |            NAME             | STATUS | VERSIONS |
++----------------------+-----------------------------+--------+----------+
+| bj03g89hpuo10ed1riog | io.anbox-cloud:nougat:amd64 | active | 1        |
++----------------------+-----------------------------+--------+----------+
+| bj003kphpuo2bt5edjtg | default                     | active | 1        |
++----------------------+-----------------------------+--------+----------+
+```
+
+In contrast to manually added images, synchronised ones are marked as immutable as they can't be manually updated. In case you want to remove an immutable image, call the following command:
+
+    amc image delete --force io.anbox-cloud:nougat:amd64
+
+If you're not using `--force`, the command will fail.
+
+## Add a new image to AMS
```

Comment:
Considering manual upload is rarely bound to happen, I think we should either drop this section or move it down, as it's less important than the rest

Reply:
I made a new section for manual upload now and moved the "Add" and "Update" sections in there. I assume the "Default image" section applies in general, thus also for the image server? How about the "Delete" section - can you delete images/versions of images provided by the image server, or only those that you upload manually?

Reply:
Nice !

Default image rules apply to all images, regardless of where they come from
Deleting images is very similar with the exception that images coming from the image server are marked as `immutable`. This means that deleting them requires the `--force` flag. Aside from that, they're the same

Reply:
Oh right - that is actually documented, but in the wrong location. Moved it to the delete section now.